<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(238, 236, 239);</string>
  </property>
  <widget class="QWidget" name="widget1" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>640</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget1 {background-color: rgb(147, 205, 255);};

</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 239, 115);
color: rgb(0, 0, 0);
font: 75 40pt &quot;Product Sans&quot;;
font-weight: bold;
</string>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="goButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>580</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#goButton {background-color: rgb(255, 239, 115);
font: 500 17pt &quot;Product Sans&quot;;
border-radius: 10px;}
</string>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="inputButton_1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Product Sans&quot;;
border-radius: 4px;
color: rgb(96, 99, 103);
background-color: rgba(255, 255, 255, 0);
</string>
    </property>
    <property name="text">
     <string>File</string>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="inputButton_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Product Sans&quot;;
background-color: rgba(255, 255, 255, 0);
border-radius: 4px;
color: rgb(96, 99, 103);
</string>
    </property>
    <property name="text">
     <string>Keyboard</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QLineEdit" name="inputFileField">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>281</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 2px;
padding-left: 10px;
color: rgb(116, 116, 118);
</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="keyField">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>281</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 2px;
padding-left: 10px;
color: rgb(116, 116, 118);
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputKeyboardField">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>401</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 2px;
padding-left: 10px;
color: rgb(116, 116, 118);
</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="inputFileButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>160</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 500 15pt &quot;Product Sans&quot;;
border-radius: 10px;
background-color: rgb(255, 239, 115);


</string>
    </property>
    <property name="text">
     <string>Choose</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="outputFileField">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>281</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 2px;
padding-left: 10px;
color: rgb(116, 116, 118);
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="outputFormatField">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>310</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 2px;
padding-left: 10px;
color: rgb(116, 116, 118);
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>320</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 15pt &quot;Product Sans&quot;;
color: rgb(116, 116, 118);</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>171</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 15pt &quot;Product Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Input Method</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>171</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 15pt &quot;Product Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Ciphertext File</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>171</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 15pt &quot;Product Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Ciphertext</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>171</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 15pt &quot;Product Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Key</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>181</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 15pt &quot;Product Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Output File</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#backButton {background-color: rgb(255, 239, 115);
font: 500 12pt &quot;Product Sans&quot;;
border-radius: 10px;}
</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="outputField">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>450</y>
      <width>401</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 2px;
padding-left: 10px;
color: rgb(116, 116, 118);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 15pt &quot;Product Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Plaintext</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>510</y>
      <width>561</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 16pt &quot;Product Sans&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
